source/tutorials_installation.md

```diff
@@ -0,0 +1,396 @@
+
+First, create an object storage bucket:
+```bash
+$ gsutil mb gs://odahu-flow-test-store/
+```
+
+Create namespace for Fluentd:
+```bash
+$ kubectl create namespace fluentd
+```
+
+Install Fluentd with specified values. Full list of values you can see in `values.yaml` from [`odahu-flow-fluentd` Helm chart repository]().
+
+Example:
+```bash
+$ cat << EOF > fluentd_values.yaml
+output:
+  target: gcs
+  gcs:
+    authorization: keyfile
+    bucket: odahu-flow-test-store
+    project: my-gcp-project-id-zzzzz
+    private_key_id: 0bacc0b0caa0a0aacabcacbab0a0b00ababacaab
+    private_key: -----BEGIN PRIVATE KEY-----\nprivate-key-here\n-----END PRIVATE KEY-----\n
+    client_email: service-account@my-gcp-project-id-zzzzz.iam.gserviceaccount.com
+    client_id: 000000000000000000000
+    auth_uri: https://accounts.google.com/o/oauth2/auth
+    token_uri: https://oauth2.googleapis.com/token
+    auth_provider_x509_cert_url: https://www.googleapis.com/oauth2/v1/certs
+    client_x509_cert_url: https://www.googleapis.com/robot/v1/metadata/x509/service-account%40my-gcp-project-id-zzzzz.iam.gserviceaccount.com
+EOF
+
+$ helm install odahu/odahu-flow-fluentd --name fluentd --namespace fluentd --values ./fluentd_values.yaml
+```
+
+## Install ODAHU
+
+### Install core ODAHU services
+
+Create namespace for core ODAHU service:
+```bash
+$ kubectl create namespace odahu-flow &&\
+    kubectl label namespace odahu-flow project=odahu-flow
+```
+
+Create namespaces for ODAHU training, packaging and deployment.
+
+```bash
+$ for i in training packaging deployment; do \
+    kubectl create namespace odahu-flow-${i} &&\
+    kubectl label namespace odahu-flow-${i} project=odahu-flow; done
+```
+
+Deployment namespace should be also labeled for Istio injection.
+
+```bash
+$ kubectl label namespace odahu-flow-deployment istio-injection=enabled
+```
+
+Prepare YAML config with values for [`odahu-flow-core`](https://github.com/odahu/odahu-flow/tree/develop/helms/odahu-flow-core) Helm chart.
+
+Example:
+```bash
+$  cat << EOF > odahuflow_values.yaml
+logLevel: debug
+ingress:
+  enabled: true
+  globalDomain: odahu.example.com
+edge:
+  ingress:
+    enabled: true
+    domain: odahu.example.com
+feedback:
+  enabled: true
+config:
+  common:
+    external_urls:
+    - name: Documentation
+      url: https://docs.odahu.org
+  connection:
+    enabled: true
+    decrypt_token: somenotemptystring
+    repository_type: kubernetes
+  deployment:
+    edge:
+      host: http://odahu.example.com
+EOF
+```
+
+
+**Note**:
+> This example uses hostname `odahu.example.com` as entrypoint for cluster services.
+>
+> It should point to LoadBalancer IP from [ingress section](tutorials_installation.html#install-nginx-ingress).
+
+In order to setup ODAHU services along with ready-to-use [connections](ref_connections.html), you may add according section to values YAML in advance:
+
+```
+connections:
+- id: docker-hub
```

Comment:
Provide an example of GCS connection for model trainings

Reply:
Done in 825ffa427aacb6fea93ba809de132546b3eb64b4